### DISCUSSION TITLE
Multi pin selection for softbody

### DISCUSSION THREAD

**Discussion by TomerconDevelopers** [godotengine/godot-proposals#6590] [3D]
When assigning pins to softbody vertices, manually selecting all pins is time consuming and nobody likes it at all.
Proposed Idea : A dedicated set of tool menu should visible when softbody node active. The menu contains bulk selection, bulk erase, bulk assign attachments etc..

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot-proposals/issues/5822.
